Source/SelfService/Web/microservice/podLogScreen.tsx

```diff
@@ -36,24 +19,17 @@ export const PodLogScreen: React.FunctionComponent = () => {
 
     return (
         <>
-            <Typography variant='h1' my={2}>Pod View Screen</Typography>
-
-            <Stack tokens={stackTokens}>
-                <Text variant="xLarge" block>
-                    application: {applicationId}
-                </Text>
-                <Text variant="xLarge" block>
-                    podName: {podName}
-                </Text>
-
-                {data.logs !== '' && (
-                    <>
-                        <pre style={{ whiteSpace: 'pre' }}>
-                            {data.logs}
-                        </pre>
-                    </>
-                )}
-            </Stack >
+            {!data.logs && (
+                <Typography variant="body2" sx={{ pl: 7.5, py: 1 }}>
+                    There are no logs printed for this microservice.
+                </Typography>
+            )}
+
+            {data.logs && (
+                <pre style={{ padding: '0.5rem 0.625rem 1.1875rem 3rem', margin: '0', whiteSpace: 'pre-wrap', wordWrap: 'break-word' }}>
+                    {data.logs}
+                </pre>
+            )}
         </>
```

Comment:
I think this would be easier to read with an if-statement like this:
```typescript
    if (!data.logs) {
        return (
            <Typography variant="body2" sx={{ pl: 7.5, py: 1 }}>
                There are no logs printed for this microservice.
            </Typography>
        );
    }

    return (
        <pre style={{ padding: '0.5rem 0.625rem 1.1875rem 3rem', margin: '0', whiteSpace: 'pre-wrap', wordWrap: 'break-word' }}>
            {data.logs}
        </pre>
    );
```